homeassistant/components/garages_amsterdam/sensor.py

```diff
@@ -32,7 +32,7 @@ async def async_setup_entry(
     async_add_entities(
         GaragesAmsterdamSensor(coordinator, entry.data["garage_name"], info_type)
         for info_type in SENSORS
-        if getattr(coordinator.data[entry.data["garage_name"]], info_type) != ""
+        if getattr(coordinator.data[entry.data["garage_name"]], info_type) is not None
```

Comment:
Why do we use getattr here btw?

Reply:
As I mentioned on Discord, the integration code is quite outdated and needs an update :S

Reply:
Ehm, this was not a question to change something, but a question to question something. But i see why we did this